admin-dev/themes/new-theme/js/pages/order/create/product-manager.js

```diff
@@ -219,8 +219,10 @@ export default class ProductManager {
   _selectFirstResult() {
     this._unsetProduct();
 
-    if (this.products.length !== 0) {
-      this._selectProduct(this.products[0].productId);
+    const keys = Object.keys(this.products);
```

Comment:
Why not taking directly the `Object.values` and looking for the first result? :thinking:

Reply:
Yeah, why not? 😅

Reply:
Done